pkg/indexed_blob/lib/indexed_blob.dart

```diff
@@ -143,13 +154,11 @@ class IndexedBlobBuilder {
 
     // Add index
     w.addKey('index');
-    _addMap(w, _index);
+    _addMap(w, index);
 
     w.endObject();
 
-    final bytes = utf8.encoder.convert(out.toString());
-
-    return BlobIndex.fromBytes(bytes);
+    return utf8.encoder.convert(out.toString());
```

Comment:
```suggestion
    return utf8.encode(out.toString());
```

Reply:
Keeping the original code as it gets a proper Uint8List, while the other one needs casting.